javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -734,6 +734,42 @@ public static ExecutableElement getMethod(
     }
 
     /**
+     * Returns the ExecutableElement for a method declaration of methodName, in class typeName, with
+     * parameters of the given types.
```

Comment:
How about adding `Errs if there is no matching method.` to make explicit that the method never returns null.